client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -193,14 +199,17 @@ export const PrescriptionLineEditForm: React.FC<
       return;
 
     const quantity = inputUnitQuantity === undefined ? 0 : inputUnitQuantity;
-    setIssueUnitQuantity(quantity);
+    const qtyWithDoses = displayInDoses ? quantity / item?.doses : quantity;
+    setIssueUnitQuantity(qtyWithDoses);
 
     const packSize =
       packSizeController.selected?.value !== -1
         ? (packSizeController.selected?.value ?? 1)
         : 1;
 
-    const numPacks = quantity / packSize;
+    const numPacks = displayInDoses
+      ? quantity / item?.doses
+      : quantity / packSize;
```

Comment:
Num packs should always relate to pack size! Don't think doses should come into play here... per above